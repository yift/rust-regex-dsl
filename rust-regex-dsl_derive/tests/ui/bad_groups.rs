pub use rust_regex_dsl::regex_dsl;

fn main() {
    let regex = regex_dsl!(
        group
    );
    println!("{}", regex);
    let regex = regex_dsl!(
        group {

        }
    );
    println!("{}", regex);
    let regex = regex_dsl!(
        group {
            nameo,
            "test"
        }
    );
    println!("{}", regex);
    let regex = regex_dsl!(
        group {
            name, "terst",
            "test"
        }
    );
    println!("{}", regex);
    let regex = regex_dsl!(
        group {
            name: "terst",
            "test"
        }
    );
    println!("{}", regex);
}