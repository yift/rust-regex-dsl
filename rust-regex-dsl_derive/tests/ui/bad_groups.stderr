error: Unknown word: group
 --> tests/ui/bad_groups.rs:5:9
  |
5 |         group
  |         ^^^^^

error: unexpected end of input, Nothing to capture in the group
  --> tests/ui/bad_groups.rs:11:9
   |
11 |         }
   |         ^

error: Unknown word: nameo
  --> tests/ui/bad_groups.rs:16:13
   |
16 |             nameo,
   |             ^^^^^

error: expected `:`
  --> tests/ui/bad_groups.rs:23:17
   |
23 |             name, "terst",
   |                 ^

error: expected identifier
  --> tests/ui/bad_groups.rs:30:19
   |
30 |             name: "terst",
   |                   ^^^^^^^
